apps/roam/src/styles/styles.css

```diff
@@ -114,6 +114,17 @@
   border-bottom: 1px solid rgba(16, 22, 26, 0.15);
 }
 
+.bp3-callout.invert-icon .bp3-icon {
+  color: #e1e8ed !important; /* @light-gray3 */
+  background-color: #106ba3; /* @blue2 */
+  padding: 1px;
+}
+
+/* Utility class that targets the width of the future popover element */
+.make-popover-full-width .bp3-popover-target {
+  width: 100%;
+}
```

Comment:
This is still ambigious (even with the comment, what is "the future popover" referring to?).  As a dev, when I see this, I now have to copy/paste the class into search and look through the repo to find it, then figure out what component it is being used for.  Alternatively, if I named this `.create-relation-dialog-popover` or something similar, I could scan this css and get the context in which it applies, without having to break flow or do extra work.